Makefile.custom.mk

```diff
@@ -0,0 +1,34 @@
+##@ Custom
+
+HELM_SCHEMA_VERSION := 0.11.1
+HELM_SCHEMA := ./bin/helm-schema-$(HELM_SCHEMA_VERSION)
+
+HELM_UNITTEST_VERSION := 0.5.0
+HELM_UNITTEST := ./bin/helm-unittest-$(HELM_UNITTEST_VERSION)
+
+.PHONY: gen-schma
+gen-schema: $(HELM_SCHEMA) ## Generates schema.
+	$(HELM_SCHEMA) -l debug
+
+.PHONY: unit-test
+unit-test: $(HELM_UNITTEST) ## Run Helm chart unit tests.
+	$(HELM_UNITTEST) ./helm/ingress-sla-app
```

Comment:
It would be good to add CI jobs for that, but I'll add it in a separate PR. This one is big enough without it.